pynicotine/slskmessages.py

```diff
@@ -2514,13 +2514,16 @@ def parse_result_bitrate_length(cls, filesize, attributes):
         # Ignore invalid values
         if bitrate <= 0 or bitrate > UINT32_LIMIT:
             bitrate = 0
-            h_bitrate = ""
+            h_quality = ""
+
+        elif sample_rate and bit_depth:
+            h_quality = f"{sample_rate / 1000:.4g} kHz / {bit_depth} bit"
 
         else:
-            h_bitrate = f"{bitrate}"
+            h_quality = f"{bitrate} kbps"
```

Comment:
kbps -> Kbps

Reply:
Maybe it's worth checking what other applications do. I'm just used to kbps for audio.